### ISSUE TITLE
Alyx plugin not working

### ISSUE THREAD

**Issue by RuyG-O** [int-brain-lab/iblrig#442]
We recently assembled an IBL behavior rig following the instructions from Appendix 3. The hardware seems to be up and running and we have installed a local instance of the Alyx database, which is currently served in our local network. However, the ‘Connect to Alyx’ option is not available when right clicking on the Project’s name in the pybpod GUI. We have verified that the module is installed in the virtual environments and that the plugin is included in the GENERIC_EDITOR_PLUGINS_LIST (see attached screenshots), but the plugin does not seem to be loaded by the pybpod GUI.

We are using the pybpod API v1.8.2

Would you have any suggestions on how to fix the plugin or an alternative strategy to make the rig talk to our local instance of Alyx?

Screenshots:

![pybpod_Alyx_plugin_issue_screenshot_1](https://user-images.githubusercontent.com/126488527/221622674-04e428f8-c1cf-4626-a029-f813a30e2048.png)
![pybpod_Alyx_plugin_issue_screenshot_2](https://user-images.githubusercontent.com/126488527/221622690-ee5ec792-65a4-48d4-9881-3156ddae981c.png)
![pybpod_Alyx_plugin_issue_screenshot_3](https://user-images.githubusercontent.com/126488527/221622695-59e07810-60b9-4db3-ac5d-c0f2ed92f02c.png)
![pybpod_Alyx_plugin_issue_screenshot_4](https://user-images.githubusercontent.com/126488527/221622698-0970958e-51b9-4719-9af4-ec4064b91d43.png)

**Comment by oliche** [comment]
Hello, do you have an error message when it fails to connect ?

**Comment by RuyG-O** [comment]
Hello,

Yes, below is a screenshot of an error message that appears on the command prompt after the software crashes. However, we can't even try to connect to Alyx because the option does not appear when we right-click on the project name in the pybpod GUI. 

![pybpod_Alyx_plugin_issue_screenshot_5](https://user-images.githubusercontent.com/126488527/222499045-8c5db5d1-84ad-4179-841e-3093b36c61f0.png)

**Comment by oliche** [comment]
Hello,

I suspect it is a ONE installation and configuration issue.

I would try to instantiate ONE in a python shell using the following recipe, with you own database and proper credentials.
https://int-brain-lab.github.io/ONE/one_installation.html#connecting-to-specific-database-relevant-for-ibl-users

Also, it seems you are on a quite old release 6.6.3, but I don't think the core of the issue is related to this

**Comment by RuyG-O** [comment]
Hello,

Thanks a lot for the help. We stopped using the old release and installed iblrig v7 instead. We also configured ONE using the recipe that you suggested with our local database and proper credentials. We are now able to log in to Alyx using the plugin in the pybpod GUI:

![image](https://user-images.githubusercontent.com/126488527/229861003-eab51ceb-1380-4225-b262-4d1e0e67288a.png)

However, when we click on 'Get Subjects' the software crashes and the following message appears on the prompt:

![image](https://user-images.githubusercontent.com/126488527/229861793-d3be2410-346a-4140-a175-f582da12e715.png)
